app/src/main/java/io/github/droidkaigi/confsched2017/model/Session.java

```diff
@@ -87,26 +86,25 @@
     private enum Type {
         CEREMONY, SESSION, BREAK;
 
-        @Override
-        public String toString() {
-            return super.toString().toLowerCase();
+        boolean matches(String type) {
+            return name().equalsIgnoreCase(type);
```

Comment:
`equalsIgnoreCase` depends on device's Locale, so can you use `name().toLowerCase(Locale.US).equals(type.toLowerCase(Locale.US))`?